templates/classify.sh

```diff
@@ -1,13 +1,8 @@
 #!/bin/bash
-if [ -z "${TMPDIR}" ]; then
-  tmp_dir="$(mktemp -d)"
-else
-  tmp_dir="${TMPDIR}"
-fi
```

Comment:
TMPDIR is initialized in nextflow.config: `TMPDIR="\${TMPDIR:-\${NXF_TEMP:-\$(mktemp -d)}}"`
TMPDIR is added to the Singularity binds in nextflow.config: `singularity_annotsv = "singularity exec --bind \${TMPDIR}`. the backslash before `${TMPDIR}` takes care that `${TMPDIR}` is evaluated per task and not on the headnode (otherwise you would get /tmp on a gearshift compute node instead of /local/<job_id>.